data/changelog.txt

```diff
@@ -1269,6 +1286,19 @@ Continue with destination activity if route is empty
 Allow moving out of impassable, aquatic treated as swimming, non-swimmers can walk in shallow water
 Book recipes are now available through camp crafting
 Fix ranged_damage automatically extending
+Hacksaw requires tempering, screwdrivers require forging
+Remove consumed faction camp calorie stores instead of leaving zombie entries
+Special vision save failure
+Fix extra handling modifier from folding stocks
+[MoM] Fix visitor pass revealing roads
+[Bugfixes] Star vampires no longer drain blood when doing no damage
+Resolve several minor Aftershock Json bugs
+Update AR-15 platforms to use new gun copy-from system
+Players learning from NPCs while blind and teaching NPCs while blind.
+Attempt to increase average autotravel speed when driving on straight roads
+Fix length for the electric lighter
+Audit "Cut slug"
+remove landmines from weird places
```

Comment:
```suggestion
Remove consumed faction camp calorie stores instead of leaving zombie entries
Special vision save failure
[Bugfixes] Star vampires no longer drain blood when doing no damage
Players learning from NPCs while blind and teaching NPCs while blind is now fixed.
Attempt to increase average autotravel speed when driving on straight roads
remove landmines from weird places
```